src/AssetBundle.php

```diff
@@ -49,6 +54,24 @@ class AssetBundle
      */
     public array $cssOptions = [];
 
+    /**
+     * @var array the options line command from converter.
+     *
+     * Example: Dart SASS minify css.
+     *
+     * public array $converterOptions = [
+     *     'css'   => '--style = compressed',
+     * ];
+     */
+    public array $converterOptions = [
```

Comment:
Yes, the AssetConverter does not provide bundle options, it must be configured globally, but if i want to only generate a css map to a specific bundle, or I want to minify a bundle, i cannot do it because everything is done globally, in my opinion using the converter from the asset manager customizing its execution would be a good improvement.